examples/tree_2d_dgsem/elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl

```diff
@@ -89,7 +89,7 @@ basis = LobattoLegendreBasis(3)
 
 limiter_idp = SubcellLimiterIDP(equations, basis;
                                 positivity_variables_cons = [
-                                    (i + 3 for i in eachcomponent(equations))...,
+                                    ("rho" * string(i) for i in eachcomponent(equations))...,
```

Comment:
```suggestion
                                    "rho" * string(i) for i in eachcomponent(equations)
```
Maybe slightly easier to read?